slick/src/test/scala/geotrellis/slick/TestDatabase.scala

```diff
@@ -3,15 +3,15 @@ package geotrellis.slick
 import com.typesafe.config.{ConfigFactory,Config}
 import org.scalatest._
 
-import scala.slick.driver.PostgresDriver.simple._
+import slick.driver.PostgresDriver.api._
 
 object TestDatabase {
   def newInstance = {
     val config = ConfigFactory.load
-    val pguser = config.getString("db.user")
-    val pgpass = config.getString("db.password")
-    val pgdb = config.getString("db.database")
-    val pghost = config.getString("db.host")
+    val pguser = "postgres"
+    val pgpass = "postgres"
+    val pgdb = "travis_ci_test"
+    val pghost = "localhost"
```

Comment:
No reason for that. I'm just not sure where I can find configuration file? Or I should add one?